subprojects/common/report-viewer/src/main/kotlin/com/avito/report/ReportsApiFactory.kt

```diff
@@ -25,7 +25,7 @@ object ReportsApiFactory {
         host: String,
         httpClientProvider: HttpClientProvider,
         loggerFactory: LoggerFactory,
-        retryRequests: Boolean = false
+        retryRequests: Boolean = true
```

Comment:
I realized that we retry never, but I think we should
because current amount of failures because of retries absence